stolostron/backlog#18755 upd hypershift (#3613)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes_intro.adoc

[#hosted-control-plane-intro]
= Using hosted control plane clusters (Technology Preview)

> {product-title} version 2.5 and multicluster engine version 2.0 can deploy {ocp} clusters using two different control plane configurations. The standalone configuration uses a virtual machine or physical machine to host the {ocp-short} control plane. Starting with {product-title-short} version 2.5, you can provision hosted control planes. By provisioning hosted control planes, you provision the {ocp-short} control plane as pods on a hosting service cluster. 

Reviewer: ```suggestion
{product-title} version 2.5 and multicluster engine version 2.0 can deploy {ocp} clusters using two different control plane configurations. The standalone configuration uses multiple dedicated virtual machines or physical machines to host the {ocp-short} control plane. Starting with {product-title-short} version 2.5, you can provision hosted control planes. By provisioning hosted control planes, you provision the {ocp-short} control plane as pods on a hosting service cluster without the need for dedicated physical machines for each control-plane.
```

> {product-title} version 2.5 and multicluster engine version 2.0 can deploy {ocp} clusters using two different control plane configurations. The standalone configuration uses multiple dedicated virtual machines or physical machines to host the {ocp-short} control plane. Starting with {product-title-short} version 2.5, you can provision hosted control planes. By provisioning hosted control planes, you provision the {ocp-short} control plane as pods on a hosting service cluster without the need for dedicated physical machines for each control-plane.

Reviewer: ```suggestion
{product-title} version 2.5 and multicluster engine version 2.0 can deploy {ocp} clusters using two different control plane configurations. The standalone configuration uses multiple dedicated virtual machines or physical machines to host the {ocp-short} control plane. Starting with {product-title-short} version 2.5 and multicluster engine version 2.0, you can provision hosted control planes. By provisioning hosted control planes, you provision the {ocp-short} control plane as pods on a hosting service cluster without the need for dedicated physical machines for each control-plane.
```

The control plane is run as pods that are contained in a single namespace and is associated with the hosted control plane cluster. When {ocp-short} provisions this type of hosted cluster, it provisions its worker node independent of the control plane. For {product-title-short}, Amazon Web Services is supported as a Technology Preview.

Some of the benefits of hosted control plane clusters include:

> * Saves cost by removing the need to host independent control plane nodes.

Reviewer: ```suggestion
* Saves cost by removing the need to host dedicated control plane nodes.
```

> * Creates a distinct division of control between the control plane and the worker nodes. The control plane can run in one account or cloud, while the workers can run in a different one.

Reviewer: ```suggestion
* Introduce separation of concerns between control plane and workloads, improving isolation and reducing misconfiguration change.
```

Author: @zanetworker I am not clear about what this bullet is saying. I've added an update that I think means the same thing.
Specific questions:
1. What does "concerns" mean in this context? Are we saying they are on different accounts or clouds?
2. What is "misconfiguration change"

Reviewer: No conerns here is more related to management and configuration. Different personas, a developer or cluster admin should not concern themselves with control-planes.

> * Decreases provisioning time of a cluster to approximately 10 minutes. 

Reviewer: ```suggestion
* Threefold decrease in cluster provisioning time as control-plane node bootstrapping is no longer required.
```

Author: Reworded this based on your suggestion.

> * Provides highly available upgrades.

Reviewer: Not needed

> * Supports cross platform hosting, in which the control plane can run in a different provider than the workers.

Reviewer: Not supported in the tech-preview so let's remove it.
